### PR TITLE
pico3 controllers

### PR DESCRIPTION
**Pull request by CtrlBrk**
I messed up with git on my side, so here's a new commit for pico3, with the controllers models this time

### CODE REVIEW

File: app/src/openxr/cpp/OpenXRInputMappings.h [hunk 1]
```diff
@@ -215,6 +215,31 @@ namespace crow {
             },
     };
 
+    const OpenXRInputMapping PicoNeo3 {
+            "/interaction_profiles/pico/neo3_controller",
+            IS_6DOF,
+            "vr_controller_pico3_left.obj",
+            "vr_controller_pico3_right.obj",
+            device::PicoNeo3,
+            std::vector<OpenXRInputProfile> { "pico-3", "generic-trigger-squeeze-thumbstick" },
```
Comment (svillar) [inline app/src/openxr/cpp/OpenXRInputMappings.h:224]: I don't think this is correct. The correct profile name is pico-neo3. See https://github.com/immersive-web/webxr-input-profiles/blob/main/packages/registry/profiles/pico/pico-neo3.json

**Review by svillar — CHANGES_REQUESTED** [review]
Many thanks for the patch! We're receiving a Neo3 soon I think so we'd be able to check first hand that this works as expected.

Patch looks mostly good but there are some more files that you should edit like ExternalVR.cpp for examples. Check for the usage of device::PicoXR for example, that should give you a good idea of what needs to be modified.

In the meantime have you verified that  the mapping works correctly in WebXR? Check what happens in this example https://immersive-web.github.io/webxr-samples/controller-state.html?usePolyfill=0

**Comment by CtrlBrk** [comment]
indeed.. I messed up so much with git that some modified files where not included ! sorry for that.
but if you're about to get a pico3 I'd better let you implement it yourself, it will be faster and easier than checking my mistakes ;)

**Comment by svillar** [comment]
> indeed.. I messed up so much with git that some modified files where not included ! sorry for that. but if you're about to get a pico3 I'd better let you implement it yourself, it will be faster and easier than checking my mistakes ;)

Not at all, we're pretty happy to receive external contributions. Experienced developers do also need to adjust their patches so don't worry about that.

Anyway, let's wait until we receive the units and then we can work on completing the PR. Thanks!

**Comment by svillar** [comment]
Closing as we added Pico3 controllers support in #1182

**Pull request closed without merging**